workflow-runtime/src/test/java/com/squareup/workflow1/internal/SubtreeManagerTest.kt

```diff
@@ -79,6 +82,7 @@ internal class SubtreeManagerTest {
     override fun snapshotState(state: Unit): Snapshot {
       snapshots++
       return Snapshot.write {
+        it.writeByte(0)
```

Comment:
What's this for?

Reply:
w/o that initialState() gets called with a null Snapshot.